src/components/section/section.md

```diff
@@ -2,15 +2,19 @@
 
 For rows that should contain more than one full-length component, a `SectionRow` should be used to wrap thse children.
 
-```
+```js
 <Section
   title="I am a section"
   description="Test description"
 />
 ```
 
 `Section`s can contain child elements, like any other element:
-```
+
+```js
+import CustomFieldInputText from '../custom-field-input-text/custom-field-input-text.jsx';
```

Comment:
Do we not get every component globally anymore?

Reply:
Correct!